eng/pipelines/aggregate-report.yml

```diff
@@ -0,0 +1,48 @@
+trigger: none
```

Comment:
Both java and python have file and pipeline called `aggregate-reports` can you please update to match?

Reply:
I will update the name.